### ISSUE TITLE
[TW-490] create new task with reverse dependency

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#525]
_Uli Martens on 2011-04-26T17:21:40Z says:_ 

 Often, I have a task in my task list and need to create a new task which has a dependency on it from the existing task. Right now I have to do this:

<pre>
$ task list
ID Project Pri Due Active Age    Description
 1                        4 secs bake a cake
$ task add buy milk
Created task 2.
$ : look for the new task ID in the above output, needed below
$ task dep:2 1
Modified 1 task
$task unblocked
ID Deps Project Pri Due Active Age   Description
 2                             1 min buy milk
$ 
</pre>

It would be much easier to have a new virtual argument "revdep" like this:

<pre>
$ task
ID Project Pri Due Active Age    Description
 1                        3 mins bake a cake
$ task add buy milk revdep:1
Created task 2.
Modified task 1.
$ task unblocked
ID Deps Project Pri Due Active Age   Description
 2                             1 min buy milk
$ 
</pre>

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2011-04-26T17:21:40Z
Modified: 2014-02-09T01:54:33Z
```

**Comment by taskwarrior** [comment]
_Cory Donnelly on 2012-03-13T16:36:50Z says:_ 

 Thanks for the error report! To keep our list of open issues manageable I'm going to go ahead and close this one as a duplicate.
